modules/installation-supported-aws-machine-types.adoc

```diff
@@ -361,3 +361,87 @@ The following Amazon Web Services (AWS) instance types are supported with {produ
 
 |===
 ====
+
+.Instance types for arm64 machines
```

Comment:
Same comment as above - question about referring to a machine by the architecture.